bitnami/common/templates/_images.tpl

```diff
@@ -21,20 +21,21 @@ Return the proper image name
 
 {{/*
 Return the proper Docker Image Registry Secret Names
-{{ include "common.images.pullSecrets" ( dict "images" (list .Values.path.to.the.image1, .Values.path.to.the.image2) "global" .Values.global) }}
+{{ include "common.images.pullSecrets" ( dict "images" (list .Values.path.to.the.image1, .Values.path.to.the.image2) "context" $) }}
```

Comment:
I'm concerned of the major change that this would imply. Couldn't we make it backwards compatible. My opinion:

Instead of substituting global with context, I would add context as a third parameter. If "context" is set, then you could use tplvalues.render. If context is not set, then use the values as they are. This way, it would be backwards compatible and we wouldn't need to bump the major in all the charts that use common. How does that sound?

Reply:
I really don't know if it's a good practice in Helm to use optional parameters. Moreover global values can be retrieved from the context anyway, so those parameters seems almost duplicates. In this case, why not create another function? Is it any better?

Reply:
As a backwards compatibliity measure, the idea would be better. Ideally, at some point, we could go changing the calls in several charts, and then in the future get rid of common.images.pullSecrets. Maybe we could call the function `common.images.renderPullSecrets`.

Reply:
Hi, I've reverted the changes to `common.images.pullSecrets` function and I've created `common.images.renderPullSecrets`.

Reply:
As of right now I would like to have this functionality into several charts that I use. Let me know if I can update those charts to use this new function with this PR or if I have to create another one.

Reply:
Hi,

We'd rather have different PRs